pkg/util/bandwidth/linux.go

```diff
@@ -131,17 +131,18 @@ func asciiCIDR(cidr string) (string, error) {
 func (t *tcShaper) findCIDRClass(cidr string) (class, handle string, found bool, err error) {
 	data, err := t.e.Command("tc", "filter", "show", "dev", t.iface).CombinedOutput()
 	if err != nil {
-		return "", "", false, err
+		return [][]string{}, false, err
 	}
 
 	hex, err := hexCIDR(cidr)
 	if err != nil {
-		return "", "", false, err
+		return [][]string{}, false, err
```

Comment:
Ditto.

Reply:
sorry about this , i want return [][]string, but miss change findCIDRClass return list